pkg/apigee/apigee.go

```diff
@@ -189,13 +191,165 @@ func (c *Client) hostAliases(org, env, virtualHost string) ([]string, error) {
 	return hosts, nil
 }
 
+// CPSEnabledForOrg returns a bool indicating whether the requested Org has CPS enabled or not
+func (c *Client) CPSEnabledForOrg(orgName string) (bool, error) {
+	c.initDefaults()
+
+	cpsEnabled := false
+
+	orgURL := fmt.Sprintf("%sv1/organizations/%s", c.ApigeeAPIHost, orgName)
+	resp, err := c.Get(orgURL)
+	if err != nil {
+		return false, err
+	}
+
+	defer resp.Body.Close()
+
+	var rawOrg interface{}
+
+	err = json.NewDecoder(resp.Body).Decode(&rawOrg)
+
+	if err != nil {
+		fmt.Printf("Error unmarshalling response: %v\n", err)
+		return false, err
+	}
+
+	org := rawOrg.(map[string]interface{})
+	orgProps := org["properties"].(map[string]interface{})
+	orgProp := orgProps["property"].([]interface{})
+
+	for _, rawProp := range orgProp {
+		prop := rawProp.(map[string]interface{})
+		if prop["name"] == "features.isCpsEnabled" {
+			if prop["value"] == "true" {
+				return true, nil
```

Comment:
Why not just set `cpsEnabled` to `true` and break the `for` loop?  Either that or make the `return` on line 230 behave as a fall back and just be `return false, nil` and not use the `cpsEnabled` variable.